book/src/generated/typable-cmd.md

```diff
@@ -71,3 +71,4 @@
 | `:append-output` | Run shell command, appending output after each selection. |
 | `:pipe` | Pipe each selection to the shell command. |
 | `:run-shell-command`, `:sh` | Run a shell command |
+| `:commands`, `:cmds` | Run commands together, use && to sepearte them |
```

Comment:
```suggestion
| `:commands`, `:cmds` | Run commands together, use && to separate them |
```

Reply:
I like the variable substitution part of this PR but I think the command chaining part here should be split to a separate PR. I'm not sure it makes sense to add since you can already chain multiple typable commands in config. Also, `&&` would imply to me that one typable command must succeed in order for the next to be run so we would probably also need `||` and `;` separators for different behaviors matching shell. And if we're matching shell then I think this should wait for scripting support anyways.

Reply:
done